gulpfile.babel.js

```diff
@@ -29,12 +30,17 @@ gulp.task('build:static', ['clean:client'], () =>
 //Convert ES6 code in all js files in src/js folder and copy to
 //build folder as bundle.js
 gulp.task('build:client', ['clean:client'], () => compileClientJS(['./src/client/javascripts/main.js'], 'main.js', './dist-client/javascripts'));
+gulp.task('build:clientcss', [], () => compileClientCSS('./src/client/stylesheets/style.css', './dist-client/stylesheets'));
 
 gulp.task('build:server', ['clean:server'], () => compileNodeJS('src/{server,shared}/**/*.js', './dist-server'));
 
-gulp.task('build', ['build:client', 'build:server', 'build:static']);
+gulp.task('build', ['build:client', 'build:clientcss', 'build:server', 'build:static']);
 
-gulp.task('build:test-client', ['clean:test'], () => compileClientJS(['./src/test/browser/index.js'], 'index.js', './dist-test/test/browser'));
+// gulp.task('build:test-client', ['clean:test'], () => compileClientJS(['./src/test/browser/components/map.js'], 'index.js', './dist-test/test/browser'));
+
+gulp.task('build:test-client', ['clean:test'], () => gulp.src(['src/test/.setup.js', 'src/test/browser/components/map.js'])  
```

Comment:
I changed specified component into 'src/test/browser/components/*.js' like this.